apps/server/src/utils/__tests__/parserFunctions.test.ts

```diff
@@ -155,3 +171,119 @@ describe('sanitiseCustomFields()', () => {
     expect(sanitationResult).toStrictEqual(expectedCustomFields);
   });
 });
+
+describe('parseRundown() linking', () => {
+  const blankEvent: OntimeEvent = {
+    id: '',
+    type: SupportedEvent.Event,
+    cue: '',
+    title: '',
+    note: '',
+    endAction: EndAction.None,
+    timerType: TimerType.CountDown,
+    linkStart: null,
+    timeStrategy: TimeStrategy.LockDuration,
+    timeStart: 0,
+    timeEnd: 0,
+    duration: 0,
+    isPublic: false,
+    skip: false,
+    colour: '',
+    revision: 0,
+    timeWarning: 120000,
+    timeDanger: 60000,
+    custom: {},
+  };
+
+  it('returns linked events', () => {
+    const data: Partial<DatabaseModel> = {
+      rundown: [
+        // @ts-expect-error - only the necessary parts
+        {
+          id: '1',
+          type: SupportedEvent.Event,
+          skip: false,
+        },
```

Comment:
suggestion: using casting would at least make sure you dont pass wrong data. I think it is more productive, like

```suggestion
        {
          id: '1',
          type: SupportedEvent.Event,
          skip: false,
        } as OntimeEvent,
```